base_layer/wallet/src/utxo_scanner_service/utxo_scanning.rs

```diff
@@ -675,7 +675,12 @@ where TBackend: WalletBackend + 'static
         let tx_id = self
             .resources
             .transaction_service
-            .import_utxo(unblinded_output.value, source_public_key.clone(), message)
+            .import_utxo(
+                unblinded_output.value,
+                source_public_key.clone(),
+                message,
+                Some(unblinded_output.features.maturity),
```

Comment:
Here it is called `maturity`. It's much less confusing if you use the same term throughout